Lib/dis.py

```diff
@@ -515,13 +515,24 @@ def _disassemble_str(source, **kwargs):
 
 disco = disassemble                     # XXX For backwards compatibility
 
+
+# Rely on C `int` being 32 bits for oparg
+_INT_BITS = 32
+# Maximum value for a c int
+_INT_MAX = 2 ** (_INT_BITS - 1)
```

Comment:
This is when it overflows, not the maximum value.
Rather than messing around with subtracting one, I'd recommend renaming this to `_INT_OVERFLOW`.
The math is sound.

Reply:
Thanks for catching, fixed!